django/core/management/commands/test.py

```diff
@@ -47,6 +47,10 @@ def add_arguments(self, parser):
             help='Overrides the default address where the live server (used '
                  'with LiveServerTestCase) is expected to run from. The '
                  'default value is localhost:8081-8179.')
+        parser.add_argument('--tag', action='append', dest='tags',
```

Comment:
I added a commit that moves that options to `DiscoverRunner`. That would be my preference instead of getting questions asking why tags don't work for a custom runner. The documentation for the new options already appears in the section that says, "These are the options of the default test runner: `DiscoverRunner`.".